app/views/shell/_top.html.erb

```diff
@@ -7,13 +7,12 @@
     <% unless internal_navigation? %>
       <meta name="last-updated" content="<%= Time.current %>">
       <meta name="user-signed-in" content="<%= user_signed_in? %>">
-      <!-- Expermiental web monetization payment pointer for micropayments -->
-      <!-- It lets readers make micropayments to websites they visit. -->
-      <!-- This is step 1: Get live in production to test for platform-wide payment pointer. -->
-      <!-- Step 2: Allow authors to set their payment pointer so they can directly monetize their content based on visitors. -->
-      <!-- Step 3: Enable further functionality based on what we learn from this experimentation and how the ecosystem evolves. -->
-      <!-- Step 3: Enable further functionality based on what we learn from this experimentation and how the ecosystem evolves. -->
       <% if SiteConfig.payment_pointer.present? %>
+        <!-- Experimental web monetization payment pointer for micropayments -->
```

Comment:
1. Moved comment block into the `if`, so far we always displayed it but then missed the `meta` tag this was referring to.
2. Fixed typo: `Expermiental` -> `Experimental`
3. Removed duplicated step 3.